src/UsgsAstroLsPlugin.cpp

```diff
@@ -17,10 +17,10 @@
 //
 //-----------------------------------------------------------------------------
 
-#define USGSASTROLINESCANNER_LIBRARY
+#define USGS_SENSOR_LIBRARY
 
 #include "UsgsAstroLsPlugin.h"
-#include "UsgsAstroLsISD.h"
+//#include "UsgsAstroLsISD.h"

```

Comment:
This can get pulled officially in a cleanup PR. This will not compile on win without this.